### PR TITLE
Script generator: Added ability to overwrite current actions with loaded parameters

### PR DESCRIPTION
**Pull request by adalator**
### Description of work

Added ability to overwrite actions with loaded parameters in script generator. A dialog window has been added which asks users whether they want to overwrite or append after selecting parameters to load. The dialog only appears when there are already actions in the table.

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/5348

### Acceptance criteria

- [ ] If there are actions in the script generator table, then loading a parameter file produces a dialog asking the user whether to append or replace actions.
- [ ] Actions can be succesfully replaced

### Unit tests

None

### System tests

None

### Documentation
None

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/ScriptGeneratorSingleton.java [hunk 1]
```diff
@@ -655,18 +655,26 @@ public void reloadScriptDefinitions() {
 		scriptDefinitionLoader.reloadScriptDefinitions();
 	}
 	
+	public void loadParameterValues(String fileName) throws NoScriptDefinitionSelectedException, ScriptDefinitionNotMatched, UnsupportedOperationException {
```
Comment (RaiBishal) [inline base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/ScriptGeneratorSingleton.java:658]: Not used, can be removed.

**Comment by DominicOram** [comment]
Addressed review comments, also created test https://github.com/ISISComputingGroup/System_Tests_UI_E4/pull/53

**Comment by DominicOram** [comment]
Have addressed the rework, not sure if you had further comments @JamesKingWork, you tailed off a bit there?

**Comment by JamesKingWork** [comment]
> Have addressed the rework, not sure if you had further comments @JamesKingWork, you tailed off a bit there?

I thought I had deleted that last line that tailed off as I ended up just repeating myself, so don't worry about that

**Pull request merged**